src/Toggles.ts

```diff
@@ -729,10 +729,10 @@ export const toggleMaxBuyCube = () => {
     const el = DOMCacheGetOrSet('toggleCubeBuy')
     if (G['buyMaxCubeUpgrades']) {
         G['buyMaxCubeUpgrades'] = false;
-        el.textContent = 'Upgrade: 1 Level wow'
+        el.textContent = 'Upgrade: 1 Level Wow!'
```

Comment:
will this be confused with wow cubes?

Reply:
Its a Wow! cube tab. Fits theme more to have Wow!
What will people think, that they will upgrade cubes themselfs if they press button?